### PR TITLE
Create performance-policy.md

### PR DESCRIPTION
**Pull request by freijernetwerven**
performance policy init

### CODE REVIEW

**Comment by janjaap** [comment]
Rename this file to README.md

**Comment by freijernetwerven** [comment]
Are we going to add separate md files for every performance best practice in this dir? Then the readme.md should be a file which tells what to expect in this dir + guidelines for the individual files, etc. The policy file is one of the files in this dir.

**Comment by freijernetwerven** [comment]
I can add a separate readme to do just that.

**Comment by mkoek** [comment]
It is best indeed to separate as much as possible and add a readme of what to expect in that directory.

**Comment by freijernetwerven** [comment]
I will do that. Thanks

File: performance/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Performance Policy And Best Practices
+===============
+This part of the best practices repo focusses on performance. 
+
+Performance Policy
+-----
+The performance policy document describes the current policy about performance measuring in our projects.
```
Comment (janjaap) [inline performance/README.md]: Link to the document: \[performance policy document](performance-policy.md)

**Pull request merged**